students/grzegorz_bajorski/lesson_06/sum_all.py

```diff
@@ -0,0 +1,8 @@
+def sum_all(*args):
+    sum = 0
```

Comment:
Try to avoid using built-in names: https://docs.python.org/3/library/functions.html

Btw built-in function ```sum()``` can be used here as an alternative solution:
```
def sum_all(*args):
      return sum(args)
```